web_services/foxx/brightid4/db.js

```diff
@@ -514,6 +522,18 @@ function linkContextId(id, context, contextId, timestamp){
     throw 'contextId is duplicate';
   }
 
+  if (!userHasVerification(verification, id)) {
+    throw 'user is not verified for this context';
+  }
```

Comment:
This 
```suggestion
```

Let's remove this check to improve the user experience (so they don't go to the app and follow a deep link and have it fail).

Could you please also find and remove the check that requires someone to be verified to be sponsored. We don't want that for the same reason.  I put a long comment about it keybase://chat/brightid#node/4193

Reply:
We didn't have anything for sponsor and you removed that check before for sponsoring users.
This condition is removed for linking too.